### ISSUE TITLE
Frontline attack Batumi keep generating errors

### ISSUE THREAD

**Issue by omsktransmash** [shdwp/dcs_liberation#18]
Game Situation
![image](https://user-images.githubusercontent.com/7844024/44401204-d4da1f00-a588-11e8-8776-7fc1056a9f6b.png)

Error Msg
```
INFO:root:DCS Libration None
INFO:root:Using 
INFO:root:<theater.base.Base object at 0x000001FD9CC2B5C0> for <class 'dcs.task.PinpointStrike'> (0): {<class 'dcs.vehicles.Armor.MBT_T_90'>: 6, <class 'dcs.vehicles.Armor.MBT_T_80U'>: 6}
INFO:root:Generating Frontline attack (regular)
ERROR:root:<class 'ValueError'>
Traceback (most recent call last):
  File "D:\Python36\lib\tkinter\__init__.py", line 1702, in __call__
    return self.func(*args)
  File "D:\Game\DCS Liberation\ui\eventmenu.py", line 249, in start
    self.game.initiate_event(self.event)
  File "D:\Game\DCS Liberation\game\game.py", line 209, in initiate_event
    event.generate()
  File "D:\Game\DCS Liberation\game\event\event.py", line 55, in generate
    self.operation.generate()
  File "D:\Game\DCS Liberation\game\operation\frontlineattack.py", line 52, in generate
    self.armorgen.generate_vec(self.attackers, self.target)
  File "D:\Game\DCS Liberation\gen\armor.py", line 104, in generate_vec
    random.randint(FRONTLINE_CAS_PADDING, int(self.conflict.distance - FRONTLINE_CAS_PADDING)))
  File "D:\Python36\lib\random.py", line 221, in randint
    return self.randrange(a, b+1)
  File "D:\Python36\lib\random.py", line 199, in randrange
    raise ValueError("empty range for randrange() (%d,%d, %d)" % (istart, istop, width))
ValueError: empty range for randrange() (12000,-11999, -23999)
```

How to reproduce
Cannot tell exactly, but now attack on Batumi keep getting the same errors
If i do other missions, it works.
However, after that if I try to attack Batumi again, same error (different unit in error msg though, like Su-33 or T_90. Above example says T_80U )

**Comment by shdwp** [comment]
Thanks, this will be fixed in next version.

**Comment by shdwp** [comment]
Should be fixed in 1.4.
